app/scripts/controllers/preferences-controller.ts

```diff
@@ -200,6 +201,7 @@ export const getDefaultPreferencesControllerState =
       showMultiRpcModal: false,
       privacyMode: false,
       shouldShowAggregatedBalancePopover: true, // by default user should see popover;
+      dismissSmartAccountSuggestionEnabled: false,
```

Comment:
We already have the `accountUpgradeDisabledChains` also, should this be combined into that?

Do we also want to display those in the settings UI?

Reply:
yep I was also thinking about re-using this, but user can change preference -and I think this will be more a user level thing and not account level